### ISSUE TITLE
Allow # prefix for color values

### ISSUE THREAD

**Issue by jasonkuhrt** [micnncim/action-label-syncer#40]
Then we can render previews for color in vscode for example.

![image](https://user-images.githubusercontent.com/284476/75819559-e8af3580-5d68-11ea-8f39-4104fc1d9612.png)

**Comment by micnncim** [comment]
The text after `#` prefix is considered as a comment, so it's an invalid value, right?

**Comment by jasonkuhrt** [comment]
True, it would have to be quoted (by the user). The tool itself would just strip it if present.

**Comment by micnncim** [comment]
Ok, I'll support that feature.
